cpu/esp32/Makefile.dep

```diff
@@ -6,6 +6,8 @@ USEMODULE += esp_idf_driver
 USEMODULE += esp_idf_esp32
 USEMODULE += esp_idf_soc
 
+USEMODULE += ztimer_msec
+
```

Comment:
```suggestion
```
Same here. It seems that makefile dependencies are correct but we have a problem with `Kconfig`. As said, adding `ztimer_msec` by default shouldn't be the solution.

Reply:
See above.